src/bin/batch_solve.rs

```diff
@@ -0,0 +1,259 @@
+use std::{
```

Comment:
Also, why is this folder named `bin`? Just b/c you're defining a CLI here?

I think the underlying functionality for this can go in something like `src/game/utils/batch`, along with aggregate report functionality. We can define the CLI in a separate file if needed.

Reply:
Yeah, we can move the functionality to a module. I still have some design decisions to consider, as we discussed before (I think there is an issue with the notion of "batch solving" which is basically "run a bunch of individual sims" when really we are creating a library of related sims, and this library should have certain operations that can be performed on it. This should resolve a lot of the ambiguities on what to do when files do/do not exist.